Node E2E: Fix wrong permission bit for log file.

Random-Liu opened the pull request:
When creating log for logs from journald, we use `0755` which is weird to me.
This PR changes it to `0666`.

<!-- Reviewable:start -->

---

This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31769)

<!-- Reviewable:end -->

Random-Liu committed:
Fix wrong permission bit for log file.

timstclair commented on test/e2e_node/services/services.go:
```diff
@@ -238,7 +238,7 @@ func (es *e2eService) getLogFiles() {
 				glog.Errorf("failed to get %q from journald: %v, %v", targetFileName, string(out), err)
 			} else {
-				if err = ioutil.WriteFile(targetLink, out, 0755); err != nil {
+				if err = ioutil.WriteFile(targetLink, out, 0666); err != nil {
```
would 0644 be more appropriate?

Random-Liu replied:
ACK. Will do.

timstclair commented:
LGTM. You can self-apply the label after you make a decision on my comment.

Random-Liu commented:
Addressed comments, and self apply LGTM based https://github.com/kubernetes/kubernetes/pull/31769#issuecomment-243820157.

This is not required for 1.4, so mark lgtm and wait for 1.5.
